expression/builtin_string.go

```diff
@@ -1151,6 +1151,7 @@ func (b *builtinLocateBinary3ArgsSig) evalInt(row []types.Datum) (int64, bool, e
 		return 0, isNull, errors.Trace(err)
 	}
 	pos, isNull, err := b.args[2].EvalInt(row, sc)
+	// transfer the argument which starts from 1 to real index which starts from 0
```

Comment:
s/ transfer/ To transfer
at `.` at the end of this comment

same as line 1190